docs/zkapps/o1js/gadgets.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: Gadgets
```

Comment:
@barriebyron 
Check out https://github.com/o1-labs/o1js/blob/main/src/lib/gadgets/gadgets.ts

Lots of copy and examples there to use